packages/validator/src/lib/index.js

```diff
@@ -17,20 +18,17 @@ module.exports = async function(
   // or the default ruleset
   let configObject;
   let spectralResults;
-  const spectral = new Spectral({
-    computeFingerprint: dedupFunction
-  });
```

Comment:
We were duplicating too much setup code, I moved all of this into the `setup` function that is called later